d2l-sort-by.html

```diff
@@ -0,0 +1,97 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-menu.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-button-subtle.html">
+<link rel="import" href="../d2l-menu/d2l-menu.html">
+
+<dom-module id="d2l-sort-by">
+	<template strip-whitespace>
+		<style>
+			:host {
+				display: inline-block;
+			}
+		</style>
+		<d2l-dropdown-button-subtle text="[[_buttonText]]">
+			<d2l-dropdown-menu no-padding no-auto-fit id="d2l-sort-by-menu">
```

Comment:
One thing I missed in the first pass: the `my-courses` screenshot I posted put the label in a visible position here above the menu. I'll include it again for context:

![screen shot 2018-10-24 at 2 06 24 pm](https://user-images.githubusercontent.com/5491151/47455974-4315aa80-d7a1-11e8-8fc9-d0dfa8b4aed6.png)

It's the "Sort By" thing at the top. Probably a question for @cheics-d2l: since I'm assuming the end goal is to replace the my-courses usage with this component, do we want that same look here, or would we prefer to not have that visible label (there'd still be an ARIA label on the menu)?

Reply:
Right now we're leaning towards removing the visible label on the Sort dropdown active state. We will likely have some sort of header for Filters though.

The objective is to replace all sorting orders with this component :)

Reply:
If the idea of a header inside of dropdown content comes up more than once, I think it's something that should may be best handled by `d2l-dropdown` content components.  I've seen other version as well that has a tabby header thing.  I think we should aim for alignment on this.